apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/components/CareerTimelineAndRecruitment.tsx

```diff
@@ -137,8 +137,8 @@ const CareerTimelineAndRecruitment = ({
               <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "This section is similar to your traditional career timeline and describes your experiences across work, school, and life. You’ll be able to reuse this information on each application you submit on the platform, speeding up the process and ensuring that your information is always up-to-date.",
-                  id: "dAGii/",
+                    "This section is similar to your traditional resume and describes your experiences across work, school, and life. You’ll be able to reuse this information on each application you submit on the platform, speeding up the process and ensuring that your information is always up-to-date.",
+                  id: "xXEhNq",
```

Comment:
Doesn't quite match the requested new copy, need to replace the "and describes your experiences across..." bit too

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/27df1720-a158-4a11-835b-71b9ee8fceb0)

Reply:
Oops! 🤦‍♀️ Fixed in [use proper new copy for career timeline description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9011/commits/ec2a6246e0b7aa6f52737a9acdf6753f8c57554e)